numpy/core/multiarray.py

```diff
@@ -1123,6 +1124,11 @@ def packbits(a, axis=None):
     axis : int, optional
         The dimension over which bit-packing is done.
         ``None`` implies packing the flattened array.
+    order : 'big' or 'little', only the first letter is checked
+        The order of the returned bits. The default is the common standard
```

Comment:
The order refers to the list of bits in the input.

Reply:
Rename to `bitorder`?

Reply:
redid the comment and changed `order` -> `bitorder`